### ISSUE TITLE
Notion won't open at all

### ISSUE THREAD

**Issue by dmercado2001** [notion-enhancer/notion-enhancer#453]
**describe the bug**
I can't open the Notion app. I see the little dot underneath the app that shows it's open, I see it at the top part all the things about the app, but I don't see the app itself. No windows can be seen.

**expected behaviour**
I should be able to actually see the app.

**screenshots**
<img width="38" alt="Screen Shot 2021-03-23 at 9 43 40 AM" src="https://user-images.githubusercontent.com/77819039/112156026-4143b180-8bbc-11eb-8a39-05c3ea20382e.png">

<img width="366" alt="Screen Shot 2021-03-23 at 9 44 05 AM" src="https://user-images.githubusercontent.com/77819039/112156105-51f42780-8bbc-11eb-8d86-d4203a9d1ec3.png">

<img width="324" alt="Screen Shot 2021-03-23 at 9 44 26 AM" src="https://user-images.githubusercontent.com/77819039/112156163-5d475300-8bbc-11eb-9cd9-764aef7bdc65.png">

**platform**
macOS Big Sur 11.2.2 I believe (I know it's in the 11.2 something)

**Comment by ayush729p** [comment]
@dmercado2001 try the fix mentioned in #437 .

**Comment by jamesNWT** [comment]
> @dmercado2001 try the fix mentioned in #437 .

Tried this, pretty sure I followed all the steps correctly, didn't work for me. My notion version is 2.0.15 which is newer than the version that that fix was supposed to be for.

**Comment by dragonwocky** [comment]
@jamesNWT double check all the steps, make sure you saved the files you edited and restarted notion etc. the fix is confirmed to work all the way up to 2.0.16 at least.

**Comment by JWilsonMD** [comment]
I had the same issue. After hours of trying different things, I thought that maybe that simply moving the application to the trash doesn't remove all files associated with the program. So I searched on YouTube "how to completely uninstall an application on macOS" and I watched this video https://www.youtube.com/watch?v=UGXVh9c-nsM

After reinstalling the program it finally opened up and worked! 

Let me know how it goes for you :)
